test/libsolidity/smtCheckerTests/imports/import_library_2.sol

```diff
@@ -0,0 +1,28 @@
+==== Source: Address.sol ====
+pragma solidity ^0.8.0;
+library Address { function s() internal pure {} }
```

Comment:
This test case does not seem to have any free functions?

Reply:
Internal library functions are considered the same as free functions for the smtchecker.

Reply:
Still, having one with an actual free function would not hurt. We should have it covered.

Reply:
We have tests for free functions and multi sources. This one is just the one that triggers the bug, and the code is executed for both actual free functions and internal library functions.

Reply:
Added a similar test with free functions instead